### PR TITLE
Transformation utilities

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        config_file = config_dir / "_init.yaml"
+
+        with open(config_file, "r") as file:
+            config = yaml.safe_load(file)
+
+        ac = config["AccelerometerSettings"]
+
+        self.accelerometeres = []
+
+        for accelerometer in range(8):
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/acceleration_transformer.py:76]: 8 accelerometers in total or only use 8 of them in this calculation? Please help to explain in the code comment.

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+        self.distances = [
+            ac["AngularAccelerationXDistance"],
+            ac["AngularAccelerationYDistance"],
+            ac["AngularAccelerationZDistance"],
+        ]
+
+    def calibrated(self, raw: pd.DataFrame) -> pd.DataFrame:
+        ret = pd.DataFrame()
+        print(raw)
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/acceleration_transformer.py:89]: Do you still need this `print()`?

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    def xyz(self, calibrated: pd.DataFrame) -> pd.DataFrame:
+        return pd.DataFrame(
+            {
+                "accelerationX": (calibrated.accelerometer7 - calibrated.accelerometer5)
+                / self.distances[0],
+                "accelerationY": (calibrated.accelerometer2 - calibrated.accelerometer0)
+                / self.distances[1],
+                "accelerationZ": (calibrated.accelerometer4 - calibrated.accelerometer0)
+                / self.distances[2],
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/acceleration_transformer.py:105]: Please help to explain this part of calculation in code comment.

**Review by teweitsai — APPROVED** [review]
Look good. Please help to explain the calculation in details in the code comment. Thanks!

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/acceleration_transformer.py:76]: Introduced N_ACCELEROMETERS constant

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py (hunk 2 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/acceleration_transformer.py:89]: No. removed - thanks, missed that.

File: python/lsst/ts/criopy/m1m3/acceleration_transformer.py (hunk 3 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/acceleration_transformer.py]: Comment added

**Pull request merged**